client/src/CSS_Readme.md

```diff
@@ -0,0 +1,56 @@
+# Description
```

Comment:
This all looks good. Could it maybe go in the existing [client-side README](https://github.com/CotripperPlatform/CoTrip/tree/develop/client)?